pages/common/tox.md

```diff
@@ -0,0 +1,24 @@
+# tox
+
+> Automate Python testing across multiple Python versions.
+> Use tox.ini to configure environments and test command.
+
+- Run tests on all test environments:
+
+`tox`
+
+- Create tox.ini configuration:
+
+`tox-quickstart`
+
+- List available environments:
+
+`tox --listenvs-all`
+
+- Run tests on a specific environment (e.g. python 3.6):
+
+`tox -e py36`
```

Comment:
Is there a long form of the `-e` argument here? Here at tldr-pages we prefer long-form options in pages, as they make them more straight-forward for new users. For example, `ls -a` is short for `ls --all`.

Reply:
Also, py36 in tokens.

Reply:
`tox --help` shows

```
  -e envlist            work against specified environments (ALL selects all).
```

Reply:
Ah, ok. No worries :-)